programs/server/dashboard.html

```diff
@@ -1077,7 +1085,7 @@
     user = document.getElementById('user').value;
     password = document.getElementById('password').value;
 
-    let url = `${host}?default_format=JSONColumnsWithMetadata&enable_http_compression=1`
```

Comment:
Why not?

Reply:
Egh, sorry, this changed accidentally, reverted.